lms/djangoapps/support/urls.py

```diff
@@ -3,25 +3,36 @@
 """
 from django.conf.urls import url
 
-from support import views
-
 from lms.djangoapps.support.views.contact_us import ContactUsView
+from support.views.certificate import CertificatesSupportView
+from support.views.course_entitlements import EntitlementSupportView
+from support.views.enrollments import EnrollmentSupportView, EnrollmentSupportListView
+from support.views.index import index
+from support.views.manage_user import ManageUserSupportView, ManageUserDetailView
+from support.views.refund import RefundSupportView
+
+COURSE_ENTITLEMENTS_VIEW = EntitlementSupportView.as_view({
+    'get': 'list',
+    'post': 'create',
+    'put': 'update'
+})
 
 urlpatterns = [
-    url(r'^$', views.index, name="index"),
-    url(r'^certificates/?$', views.CertificatesSupportView.as_view(), name="certificates"),
-    url(r'^refund/?$', views.RefundSupportView.as_view(), name="refund"),
-    url(r'^enrollment/?$', views.EnrollmentSupportView.as_view(), name="enrollment"),
+    url(r'^$', index, name="index"),
+    url(r'^certificates/?$', CertificatesSupportView.as_view(), name="certificates"),
+    url(r'^refund/?$', RefundSupportView.as_view(), name="refund"),
+    url(r'^course_entitlement/?$', COURSE_ENTITLEMENTS_VIEW, name="course_entitlement"),
```

Comment:
I think we should consider using the url `entitlement/` rather than `course_entitlement`.  It will better match the `enrollment/` url.

Reply:
I was thinking that course_entitlement was a bit more descriptive. For `enrollment` it's already fairly clear that enrollment in a course is being referenced, but entitlement is a term that's not typically linked to course engagement.